Pegasus/sources/Integration.cpp

```diff
@@ -67,27 +67,27 @@ glm::dvec3 IntegrateLinearDamping(glm::dvec3 velocity, double damping, double du
 void IntegrateBody(
         pegasus::mechanics::Material& material,
         pegasus::mechanics::Body::LinearMotion& linearMotion,
-        double duration
+        float duration
     )
 {
     {
-        double const maxSpeed = 100;
-        double const speed = glm::length(linearMotion.velocity);
+        float const maxSpeed = 100;
```

Comment:
Is this limit mentioned somewhere? Should it be configurable?

Reply:
It is not, I'm not sure.